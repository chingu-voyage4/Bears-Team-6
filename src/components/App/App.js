import React from 'react'

const App = () => (<span>We are.</span>)

export default App
