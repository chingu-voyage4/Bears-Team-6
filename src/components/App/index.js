// @flow

export { connected as App } from './App'
