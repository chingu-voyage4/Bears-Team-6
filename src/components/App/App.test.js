import React from 'react'
import App from './App'
import {shallow} from 'enzyme'

const wrapper = shallow(<App/>);

describe('App', () => {
  it('renders as expected', () => {
    expect(wrapper).toMatchSnapshot();
  });
  it('has "We are." inside', () => {
    expect(wrapper.prop('children')).toEqual('We are.');
  });
});
