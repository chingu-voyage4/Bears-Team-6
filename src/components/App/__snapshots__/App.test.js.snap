// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders as expected 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <App />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "We are.",
    },
    "ref": null,
    "rendered": "We are.",
    "type": "span",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "We are.",
      },
      "ref": null,
      "rendered": "We are.",
      "type": "span",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
